std/io/io.neva

```diff
@@ -1,2 +1,5 @@
 #extern(scanln)
-pub component Scanln(sig any) (data string)
\ No newline at end of file
+pub component Scanln(sig any) (data string)
+
+#extern(args)
+pub component Args() (data list<string>)
```

Comment:
We decided to have create `os` package to follow Go's std structure, don't we? :)

Reply:
fixed